src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/model/response/DischargeToHospitalResponse.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.response
+
+import java.time.LocalDate
+
+data class DischargeToHospitalResponse(
+  val restrictivePatient: RestrictivePatient
+)
+
+data class RestrictivePatient(
```

Comment:
Is the name right?

Reply:
Yeah, that's the prop name in the prison api response.